backend/src/routes/api/nb-events/eventUtils.ts

```diff
@@ -0,0 +1,95 @@
+import { V1EventList } from '@kubernetes/client-node';
+import { KubeFastifyInstance, NotebookStatus } from '../../../types';
+
+export const getNotebookEvents = async (
+  fastify: KubeFastifyInstance,
+  nbName: string,
+): Promise<NotebookStatus> => {
+  const response = await fastify.kube.coreV1Api
+    .listNamespacedEvent(
+      fastify.kube.namespace,
+      undefined,
+      undefined,
+      undefined,
+      `involvedObject.kind=Notebook,involvedObject.name=${nbName}`,
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      false,
```

Comment:
`watch?: boolean` should have a non-truthy value by default. This is necessary, right? How would we ever be able to respond to the watch values in the current infrastructure. I think you can simplify this.

Reply:
Removed all four additional undefined values, is that what you meant?